pages/pipelines/notifications.md.erb

```diff
@@ -0,0 +1,58 @@
+# Triggering Notifications
+
+The `notify` attribute allows you to trigger build notifications to different services. You can also choose to conditionally send notifications based on pipeline events like build status.
+
+<%= toc %>
+
+Add notifications to your pipeline with the `notify` attribute. This sits  at the same level as `steps` in your `pipeline.yml` file.
+
+The below example will send a notification email every time a build is created on this pipeline: 
+
+```yaml
+steps:
+  - command: "tests.sh"
+  - wait
+  - command: "build.sh"
+
+notify:
+  - email: "dev@acmeinc.com"
+```
+
+You can trigger notifications as emails or as a message in a Basecamp Campfire.
+
+## Conditional notifications
+
+You can choose to only trigger notifications when a statement evaluates to true using the `if` attribute.
+
+In the below example the email notification will only be triggered if the build passes:
+
+```yaml
+notify:
+  - email: "dev@acmeinc.com"
+    if: "build.status == passed"
+```
+
+## Email
+
+Add an email notification to your pipeline using the `email` attribute of the `notify` yaml block:
+
+```yaml
+notify:
+  - email: "dev@acmeinc.com"
+```
+
+The `email` attribute accepts a single email address as a string.
+
+## Basecamp Campfire Message
+
+Add a basecamp notification to your pipeline using the `basecamp_campfire` attribute of the `notify` yaml block:
+
+```yaml
+notify:
+  - basecamp_campfire: URL
+```
+
+The `basecamp_campfire` attribute accepts a single URL as a string. The URL for your Basecamp Campfire can be created using the 'Generate a public link' button in Basecamp.
```

Comment:
This is currently only scoped to v. 3 - I'll get you an example URL!

Reply:
basically it has to have the `3.`:
```
 - basecamp_campfire: "https://3.basecamp.com/hello"
```

Reply:
Rad, thanks 👌🏻 Did you happen to find out which URL it is too? I just guessed that it was public URL thingo.

Reply:
The URL for Basecamp is the "Command URL"

![image](https://user-images.githubusercontent.com/25882/69285820-c4ea3880-0c2c-11ea-9f22-91c8dc48469b.png)

See here: https://m.signalvnoise.com/new-in-basecamp-3-chatbots/

This is the image we use in our chat if you wanted to include it in the docs as well as a "hey's a picture you can use"

![avatar-128-x1](https://user-images.githubusercontent.com/25882/69285767-a71cd380-0c2c-11ea-88a0-e49ce3a9fcd3.gif)

Reply:
@keithpitt how do you know what URL to put in there? Is it a basecamp one or a Buildkite one?

Reply:
Sorry, I lied. "Command URL" isn't what you want. Once you've created a ChatBot with a name and an avatar, the next page has a dropdown thing that you click.

![image](https://user-images.githubusercontent.com/25882/69395834-56d06f00-0d1b-11ea-8e62-808328dfa90c.png)

The URL there is the thing you copy/paste into here:

```
basecamp_campfire: "https://3.basecamp.com/.../integrations/.../buckets/.../chats/.../lines"
```

Reply:
You've got to be a basecamp admin to be able to do that though!